portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -29,24 +35,37 @@
 
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    Query query = new Query("Youtube Idea").addSort("timestamp", SortDirection.ASCENDING);
 
-    // TODO:  Make ArrayList more abstract aka make it a List.
-    ArrayList<String> sampleContents = new ArrayList<String>();
-    sampleContents.add("My name is Miah");
-    sampleContents.add("Im from NYC");
-    sampleContents.add("I love coding!");
+    DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
```

Comment:
private final instance variable for this as we only need one datastore service instance per DataServlet instance

Reply:
i did this in my post-practice branch which just got approved so I will merge that now!